src/v/pandaproxy/schema_registry/json.cc

```diff
@@ -1253,11 +1271,18 @@ make_json_schema_definition(sharded_store&, canonical_schema schema) {
 }
 
 ss::future<canonical_schema>
-make_canonical_json_schema(sharded_store&, unparsed_schema def) {
+make_canonical_json_schema(sharded_store& store, unparsed_schema def) {
     // TODO BP: More validation and normalisation
-    parse_json(def.def().raw()()).value(); // throws on error
```

Comment:
How come this line with `parse_json(...)` is removed?

Reply:
> How come this line with `parse_json(...)` is removed?

Because even after about 5 attempts at this PR, I can't get it right.

Reply:
Fixed